### DISCUSSION TITLE
有问题直接加微信群

### DISCUSSION THREAD

**Discussion by Hanson** [HanSon/vbot#320] [Announcements]
![b2d2e0a3ba65d7355bb83c4aeda9540](https://user-images.githubusercontent.com/10583423/135378596-4e650b70-3c32-4bb9-be59-7857390b544d.jpg)

如过期直接在底下说一下，定时更新

**Comment by krislwang** [comment]
码过期了

**Reply by Hanson** [reply]
目前不开放群了

**Reply by millionmax625** [reply]
怎么使用呢？二维码过期了

**Reply by Hanson** [reply]
只能商业化使用了，加微信 hansonskr
